UI/Pages/Accessibility.cshtml

```diff
@@ -29,12 +29,11 @@
             navigated with common accessibility tools.
         </p>
         <h3 class="govuk-heading-m" data-testid="access-parts-header">What to do if you cannot access parts of this website</h3>
-        <p class="govuk-body">
-            If you need information on this website in a different format like an accessible PDF, large print, easy
-            read, audio recording or braille please contact us at:
-        </p>
         <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
-            <li>tutoring.support@service.education.gov.uk</li>
+            <li>
+                If you need information on this website in a different format like an accessible PDF, large print, easy
+                read, audio recording or braille please contact us at: tutoring.support@service.education.gov.uk
```

Comment:
tutoring.support@service.education.gov.uk needs to be a link

Reply:
It wasn't specified in the ticket, but it is a good idea.